controllers/postController.js

```diff
@@ -277,4 +300,156 @@ const postCount = async (req, res) => {
   }
 };
 
-module.exports = { upsert, list, today, postCount };
+const groupPostCount = async (req, res) => {
+  if (!isValidString(req.params.groupId) || isEmptyString(req.params.groupId)) {
+    return res.status(400).send({ message: "[InvalidGroupId] Error occured" });
+  }
+
+  const groupInfo = await groupModel.findOne({ _id: req.params.groupId }).exec();
+  if (groupInfo === null) {
+    return res.status(400).send({ message: "[NotExistedGroupId] Error occured" });
+  }
+
+  const groupId = req.params.groupId;
+
+  let cursorIdDate;
+  if (isValidString(req.query.cursorId)) {
+    if (isEmptyString(req.query.cursorId)) {
+      cursorIdDate = new Date();
+      cursorIdDate.setHours(0, 0, 0, 0);
+      cursorIdDate.setDate(cursorIdDate.getDate() - 1 - cursorIdDate.getDay());
```

Comment:
```suggestion
      cursorIdDate = new Date();
      cursorIdDate.setDate(cursorIdDate.getDate() - 1 - cursorIdDate.getDay());
      cursorIdDate.setHours(0, 0, 0, 0);
      cursorIdDate = cursorIdDate.toISOString();
```
해당 코드에 이슈가 있어서 제 PR에 수정해서 올렸는데, 동일한 코드여서 수정한 코드 전달드려요!
`setHours`를 하게되면 밀리세컨드 시간으로 반환되어서, `cursorIdDate.getDate()`를 찾을 수 없다는 에러가 떠서 수정했는데 그룹 대시보드 쪽도 반영 부탁드립니다!

Reply:
말씀 주신대로 수정했습니다.